setup.py

```diff
@@ -132,6 +132,11 @@
     "flask-sockets",
     "gevent",
     "gevent-websocket",
+
+    #############################
+    # Advanced image processing #
+    #############################
+    "imutils",
```

Comment:
Is this a core dependency? How big is it?

Reply:
It's pretty useful, I find myself needing more of his stuff all the time. We could just copy and paste the stuff we need into our own lib (I did this before) but it's a nice lib that people use a lot for opencv so I think it's worth keeping. I believe it's around 50kb